tests/dataset_runner.py

```diff
@@ -61,7 +91,14 @@ def run(self, dataset_fixture, run_name_prefix="test"):
         self.wait_for_envelope_to_be_validated()
         if self.export_bundles:
             self.complete_submission()
-            self.wait_for_primary_and_results_bundles()
+            if run_name_prefix == "scale":
+                # == Scaling Logic ==
```

Comment:
I think you can also remove the `# == Scaling Logic ==` and `# == Non-scaling Logic ==` comments since the conditional statement is self-explanatory

Reply:
Will do!